### DISCUSSION TITLE
Helpers: Options should be movable

### DISCUSSION THREAD

**Discussion by flemmingss** [home-assistant/frontend#10106] [Configuration] [answered]
In the example I have 4 options, and i can't move "None" to be on the top above "Fast Colorloop".
This should be possible by drag-and-drop or by up/down arrows 

![image](https://user-images.githubusercontent.com/25674067/135401021-5ec09331-156d-4caa-96d7-f424660365fd.png)
Not the most critical request, but I think it will be nice.
Example of how it works in Lovelace (drag and drop):
![image](https://user-images.githubusercontent.com/25674067/135401794-d57c44ee-077c-4357-a37c-0aeb3da17478.png)

**Comment by matthiasdebaat** [comment]
Yeah good one! I'll make it in a Hacktoberfest issue.

**Reply by matthiasdebaat** [reply]
https://github.com/home-assistant/frontend/issues/10107

**Comment by matthiasdebaat** [comment, marked as answer]
Duplicaten of https://github.com/home-assistant/frontend/discussions/11238
